tools/clang/unittests/HLSLTestLib/D3DReflectionStrings.cpp

```diff
@@ -198,7 +198,18 @@ LPCSTR ToString(D3D_TESSELLATOR_DOMAIN TessellatorDomain) {
   default: return nullptr;
   }
 }
+
+// FIXME: remove this once D3D_SVC_BIT_FIELD added into
+// D3D_SHADER_VARIABLE_CLASS.
+const D3D_SHADER_VARIABLE_CLASS D3D_SVC_BIT_FIELD =
```

Comment:
See how values are added for 16 and 64 bit types under `ADD_16_64_BIT_TYPES`.  Maybe we should follow the same pattern.  It allows for compilation against the header after you add it as well as before.